jquery.scrolling-tabs.scss

```diff
@@ -36,20 +44,20 @@ $background-color-bootstrap-gray-hover: rgb(238, 238, 238);
 }
 
 .scrtabs-tab-scroll-arrow {
-  border: $border-bootstrap-lt-gray;
+  border: $scrtabs-border-color;
   border-top: none;
-  color: $color-bootstrap-blue;
+  color: $scrtabs-background-color-hover;
```

Comment:
Sorry, I didn't notice this before I merged&mdash;I believe this should be `$scrtabs-foreground-color`, correct?

I can just make the change. No need to submit another PR.